### PR TITLE
Fixed handler array for dynamic values

### LINKED ISSUE #190: Allow 3rd parties modifying course enrollment visibility options
In LifterLMS core with this filter: https://developer.lifterlms.com/reference/hooks/lifterlms_product_visibility_options/
you can add/remove the course visibility options.

Using that filter automatically reflects on the course meta box when using the classic editor, but it doesn't on the block editor, because that option is handled with js (react).
https://lifterlms.com/docs/course-membership-visibility-settings/

The idea would be to add the necessary js ~filter (see `applyFilters` from `@wordpres/wp-hooks`)~ code (see comment below) to allow 3rd parties to modify that option (along to the PHP filter which will be always needed).

### PR DESCRIPTION
**Pull request by actuallyakash**
## Description
Fixed static array issue with Handler object.

Fixes #190 

## How has this been tested?
Manually.

## Types of changes
Bug fix

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

File: .changelogs/issue_190.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+significance: patch
+type: fixed
+links:
+  - "#190"
+attributions:
+  - "@seothemes"
```
Comment (eri-trabiccolo) [inline .changelogs/issue_190.yml:6]:
```suggestion
attributions:
  - "@seothemes"
```
Attributions is for external contributors only :D

Comment (eri-trabiccolo) [inline .changelogs/issue_190.yml]:
```suggestion
```
Attributions is for external contributors only :D

File: .changelogs/issue_190.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+entry: Fixed conflict when values are added via filter in Visiblity Options.
```
Comment (eri-trabiccolo) [inline .changelogs/issue_190.yml]:
```suggestion
entry: Fixed conflict when values are added via filter in Visibility Options.
```

Comment (eri-trabiccolo) [inline .changelogs/issue_190.yml]:
```suggestion
entry: Fixed conflict when values are added via filter in Visibility Options.
```

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]

File: .changelogs/issue_190.yml (hunk 2 above)
Comment (actuallyakash) [reply .changelogs/issue_190.yml]: fixed

File: .changelogs/issue_190.yml (hunk 1 above)
Comment (actuallyakash) [reply .changelogs/issue_190.yml]: :D ohkay

**Review by eri-trabiccolo — APPROVED** [review]

**Pull request merged**